packages/notification/src/components/GenericNotification.tsx

```diff
@@ -192,4 +197,32 @@ const CancelButton = ({ onClick }: CancelButtonProps) => (
   </button>
 )
 
+const NotificationHeading = ({ titleProp, headingProps, style }) => {
+  if (headingProps) {
+    return (
+      <Heading
+        variant={headingProps.variant || "heading-6"}
+        tag={headingProps.tag || "div"}
+        color={headingProps.color || "dark"}
+        classNameOverride={styles.notificationTitle}
+      >
+        {headingProps.children}
+      </Heading>
+    )
+  } else if (titleProp) {
+    return (
+      <Heading
+        variant="heading-6"
+        tag={style === "toast" ? "h6" : "div"}
```

Comment:
This may be a bit controversial, changing the default from a `h6` to a `div` when no `headingProps` provided, but follows the discussion in https://github.com/cultureamp/kaizen-design-system/pull/2982 not to cause violations

Reply:
Yeah I don't think there was any good reason for this to be an `h6`, any hardcoded heading level like this in a reusable component seems like a bad idea.

Can we get rid of it for `style === toast` as well? Any particular reason you've opted to continue doing that here?

Reply:
@dougmacknz I was nervous to change the toast in case it messed up with any announcements for screen readers-  but if thats not an issue I can apply the same!

Reply:
Yeah I don't think this is helping with that, we really need a live region for toasts. For now unfortunately they're not screen reader accessible.

Reply:
Just set div as the default 👍 
@mcwinter07 as the ternary isn't there anymore I wont add the extra test